ext/nnstreamer/tensor_decoder/tensordec-pose.c

```diff
@@ -29,11 +29,15 @@
  * option1: Video Output Dimension (WIDTH:HEIGHT)
  * option2: Input Dimension (WIDTH:HEIGHT)
  * option3: Location of label file (optional)
+ * option4: Mode (optional)
+ *          Available: heatmap-only (default)
+ *                     heatmap-offset
  */
```

Comment:
https://github.com/nnstreamer/nnstreamer/blob/main/ext/nnstreamer/tensor_decoder/tensordec-imagesegment.c#L37-L75

How about write brief information about the expected tflite model and sample pipeline?

```bash
gst-launch-1.0 v4l2src ! videoconvert ! videoscale ! \
  video/x-raw,format=RGB,width=640,height=480,framerate=30/1 ! \tee name=t \
  t. ! queue ! videoscale ! video/x-raw,width=257,height=257,format=RGB ! \
  tensor_converter ! tensor_transform mode=arithmetic option=typecast:float32,add:-127.5,div:127.5 ! \
  tensor_filter framework=tensorflow-lite model=posenet_mobilenet_v1_100_257x257_multi_kpt_stripped.tflite ! \
  tensor_decoder mode=pose_estimation option1=640:480 option2=257:257 option3=pose_label.txt option4=heatmap-offset ! \
  compositor name=mix sink_0::zorder=1 sink_1::zorder=0 ! videoconvert ! ximagesink \
  t. ! queue ! mix.
```

I confirmed that the above pipeline works well with the model from tflite example and proper `pose_label.txt`.

Reply:
Thanks for your comments and test. I will amend the PR with required information.